logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/CallMapper.kt

```diff
@@ -64,10 +65,25 @@ class CallMapper {
                 clientId = clientid
             )
         }
+    }
+
+    inner class ActiveSpeakerMapper {
+        fun mapParticipantsActiveSpeaker(
+            participants: List<Participant>,
+            activeSpeakers: CallActiveSpeakers
+        ) : List<Participant> = participants.map { participant ->
+            participant.copy(
+                isSpeaking = activeSpeakers.activeSpeakers.any {
+                    it.userId == participant.id.toString()
```

Comment:
we should compare with clientId also, the same user can join a group call with different clients

Reply:
Nice! great catch